azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/service/sdk/CallingSDKWrapper.kt

```diff
@@ -262,6 +262,8 @@ internal class CallingSDKWrapper(
         return call.mute(context)
     }
 
+    private fun callIsNotSetup() = callClient == null && deviceManagerCompletableFuture == null
```

Comment:
Albert, ca we name this as something like 'canCreateLocalVideostream()`. It will be great if we can add comment to share cause of it.

Reply:
done. also added explanatory comment